### ISSUE TITLE
Icon request: sitemap

### ISSUE THREAD

**Issue by alexantr** [colebemis/feather#94]
Icon: sitemap
Showing site structure.
Example:

![sitemap](https://user-images.githubusercontent.com/4941848/26886266-94c0df98-4bad-11e7-8741-f24d3533f0b1.png)

![sitemap-256](https://user-images.githubusercontent.com/4941848/26886337-bc82a336-4bad-11e7-9cfe-9f7572bee230.png) ![sitemap-256 1](https://user-images.githubusercontent.com/4941848/26886341-be367a9a-4bad-11e7-9ffa-50982ae16acb.png)

**Comment by johnletey** [comment]
@locness3 @colebemis @alexantr How does [this](https://www.figma.com/file/nqmUUrLUh4NBxXI39XDtgp/sitemap) look? Unfortunately, I couldn't fit three squares in the bottom row ... only two

<img width="534" alt="Screen Shot 2019-07-01 at 8 43 50 PM" src="https://user-images.githubusercontent.com/30328854/60456110-1162b680-9c41-11e9-9a71-d13baef58753.png">

**Comment by locness3** [comment]
> @locness3 @colebemis @alexantr How does [this](https://www.figma.com/file/nqmUUrLUh4NBxXI39XDtgp/sitemap) look? Unfortunately, I couldn't fit three squares in the bottom row ... only two
> 
> <img alt="Screen Shot 2019-07-01 at 8 43 50 PM" width="534" src="https://user-images.githubusercontent.com/30328854/60456110-1162b680-9c41-11e9-9a71-d13baef58753.png">

Thta is great as it is. We don't need three squares; two is enough.

**Comment by locness3** [comment]
Although we should think of a different name, since this does not only represents a sitemap.

**Comment by johnletey** [comment]
> Although we should think of a different name, since this does not only represents a sitemap.

@locness3 Nothing comes to mind as to what else this represents

**Comment by locness3** [comment]
> > Although we should think of a different name, since this does not only represents a sitemap.
> 
> @locness3 Nothing comes to mind as to what else this represents

Ethernet, organisation are what I came up with

**Comment by johnletey** [comment]
> > > Although we should think of a different name, since this does not only represents a sitemap.
> > 
> > 
> > @locness3 Nothing comes to mind as to what else this represents
> 
> Ethernet, organisation are what I came up with

@locness3 Understand ... so yes, we definitely need to come up with a different name!

**Comment by alexantr** [comment]
Maybe lan or network?

**Comment by locness3** [comment]
> Maybe lan or network?

Yes, since it can be a network of anything (computers, people, ...)

**Comment by johnletey** [comment]
Great ... will call it network and open up a PR!
